### PR TITLE
tracee-ebpf: add container id to context

### PR DESCRIPTION
**Pull request by yanivagman**
Partially covers #430

### CODE REVIEW

File: tracee-ebpf/tracee/tracee.bpf.c [hunk 1]
```diff
@@ -203,6 +205,8 @@
 
 #define DEV_NULL_STR    0
 
+#define CONT_ID_LEN 12
```
Comment (simar7) [inline tracee-ebpf/tracee/tracee.bpf.c:208]: Curious why is this 12?

Comment (yanivagman) [reply tracee-ebpf/tracee/tracee.bpf.c:208]: I could have chosen a bigger value, but I don't think that this is necessary to identify a container.
Docker also presents the first 12 chars in its output so I used the same convention here.

**Review by simar7 — APPROVED** [review]

File: tracee-ebpf/tracee/printer.go [hunk 2]
```diff
@@ -114,13 +115,13 @@ func (p tableEventPrinter) Init() error { return nil }
 func (p tableEventPrinter) Preamble() {
 	if p.verbose {
 		if p.containerMode {
-			fmt.Fprintf(p.out, "%-14s %-16s %-12s %-12s %-6s %-16s %-15s %-15s %-15s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "MNT_NS", "PID_NS", "UID", "COMM", "PID/host", "TID/host", "PPID/host", "RET", "EVENT", "ARGS")
```
Comment (itaysk) [inline tracee-ebpf/tracee/printer.go:117]: @yanivagman don't you think hostname is redundant in container mode now that we have container id?

**Comment by itaysk** [comment]
@yanivagman this will not work with pre-existing container right?

**Comment by itaysk** [comment]
just sharing a thought publicly (not asking for any change), there are some similarities between this code and the "capture exec" code, in the future we should consider if we once could benefit form the other

File: tracee-ebpf/tracee/printer.go (hunk 2 above)
Comment (yanivagman) [reply tracee-ebpf/tracee/printer.go:117]: Yes, we can leave it to verbose only

**Comment by yanivagman** [comment]
`@yanivagman this will not work with pre-existing container right?`
Correct. We should handle this case differently.
A simple approach to handle this will be to update the new "pid_to_cont_id_map" map on Tracee init.
This will have a small race condition though (if a new container starts/exits while we init tracee)

**Comment by yanivagman** [comment]
> just sharing a thought publicly (not asking for any change), there are some similarities between this code and the "capture exec" code, in the future we should consider if we once could benefit form the other

Which similarities?

**Pull request merged**